app/src/components/Nav/index.jsx

```diff
@@ -15,7 +13,9 @@ const Nav = () => {
   return (
     <div className="main-menu">
       <input type="checkbox" id="checkbox-toggle" />
-      <label class="menu-title" for="checkbox-toggle">Menu</label>
+      <label className="menu-title" htmlFor="checkbox-toggle">
+        Menu
```

Comment:
I fixed the tag names with React conventions i.e. `class` -> `className` and `for` -> `htmlFor` as otherwise it was logging some errors. As a result, the length of the line got updated, exceeding the present max length in the editor, thus pushing the ending tag and the contents to new lines.

Do you have a preference for the max chars in a single line? @thisisjofrank